app/scripts/controllers/listperson.js

```diff
@@ -1,18 +1,71 @@
 'use strict';
 
-/**
- * @ngdoc function
- * @name admissionSystemApp.controller:ListpersonctrlCtrl
- * @description
- * # ListpersonctrlCtrl
- * Controller of the admissionSystemApp
- */
- 
+
 angular.module('admissionSystemApp')
-  .controller('ListPersonCtrl', function ($scope) {
-    $scope.awesomeThings = [
-      'HTML5 Boilerplate',
-      'AngularJS',
-      'Karma'
+  .controller('ListPersonCtrl', function($scope, personDecodeServise) {
+
+    $scope.dataDecoded = [];
+    personDecodeServise.personDecoded().then(function (data) {
+      $scope.dataDecoded = data;
+    });
+
+    $scope.filters = [
+      {
```

Comment:
it is just for sake of demonstration that directive and decoding service is working